source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt

```diff
@@ -54,56 +54,127 @@ Use a Consistent Sentence Structure
-       containers that are hosted within that data center. Then, the
-       app displays the number of files in each container and the
-       approximate size of each container.
-     - After you choose a data center, the app retrieves a list of the
-       containers that are hosted within that data center, along with
-       the number of files in each container and the approximate size.
+   * - After you choose a cloud provider, Atlas fetches a list of the
+       regions where the provider can host a cluster. Atlas displays the default cluster tier with its memory and storage parameters.
+     - After you choose a cloud provider, Atlas fetches a list of the
+       regions where the cloud provider can host a cluster, along with
+       the cluster tier and the memory and storage parameters for that tier.
 
    * - Select whether to overwrite files with the same name or to
-       restore files to their original folders. Click **Next**.
+       restore files to their original folders. Click :guilabel:`Next`.
      - Click the check boxes to confirm whether you would like to
        Overwrite files with the same name or restore the files to their
-       original folders and then click the **Next** button.
+       original folders and then click the :guilabel:`Next` button.
+
+.. _sg-limit-verb-be:
+
+Limit Using *Be* as the Main Verb
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pay special attention if a form of be is the main verb in the sentence.
+Forms of *be* include:
+
+- *be*
+- *is*
+- *am*
+- *are*
+- *was*
+- *were*
+- *being*
+- *been*
+
+Look for a "hidden" verb in the element following the *be*‐verb.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - This report analyzes the problem.
+     - This report is an analysis of the problem.
+
+.. _sg-avoid-expletive-structures:
+
+Avoid Expletive Structures
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An expletive sentence begins with *It* or *There* and followed by a form of *be*. Consider another noun and verb following the expletive.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - We need to solve this problem.
+     - It is necessary for us to find a solution to this problem.
+
+   * - There are three benefits to the cloud backup compared to the
+       legacy backup.
+     - Cloud backup has three benefits over legacy backup.
+
+.. _sg-avoid-nominalizations:
+
+Avoid Nominalizations
+~~~~~~~~~~~~~~~~~~~~~
+
+A *nominalization* is a noun derived from a verb. These words often end
+in -ance, -ence, -ion, or -ment. Identify the verb at the root of the
+nominalization and use it instead.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - The modal recommends upgrading to new software.
+     - The modal contains a recommendation that the user upgrade to new
+       software.
+
+   * - It would benefit the users if they had this information.
```

Comment:
Even better: "It benefits the users to have this information"

Reply:
Did this look familiar?